### DISCUSSION TITLE
Centering both axes

### DISCUSSION THREAD

**Discussion by abiriadev** [plotters-rs/plotters#460] [Q&A]
First of all, please let me know if this is not the appropriate place to ask basic questions.

And I'd like to express gratitude for creating this awesome crate! I am currently following the official example and trying to customize it as much as possible.

My question is, how can I align (both) X and Y axes to the center of the chart? I've searched the docs and investigated [MeshStyle](https://docs.rs/plotters/latest/plotters/chart/struct.MeshStyle.html) struct but couldn't find any similar method. I've looked for similar issues or discussions, but with no luck.

Below is the entire code:
```rs
use std::{error::Error, f64::consts::PI};

use plotters::{
	prelude::{
		BitMapBackend, ChartBuilder, Circle, EmptyElement, IntoDrawingArea,
		Text,
	},
	series::PointSeries,
	style::{full_palette::PURPLE, IntoFont, WHITE},
};

fn w_n(n: u32) -> Vec<(f64, f64)> {
	(0..n)
		.into_iter()
		.map(|r| {
			let (s, c) = ((2. * PI) / (n as f64) * r as f64).sin_cos();
			(c, s)
		})
		.collect()
}

fn main() -> Result<(), Box<dyn Error>> {
	let root =
		BitMapBackend::new("chart.png", (1200, 1200)).into_drawing_area();

	root.fill(&WHITE)?;

	let mut chart = ChartBuilder::on(&root)
		.margin(5)
		.x_label_area_size(30)
		.y_label_area_size(30)
		.build_cartesian_2d(-1.5f64..1.5, -1.5f64..1.5)?;

	chart
		.configure_mesh()
		.max_light_lines(0)
		.draw()?;

	chart.draw_series(PointSeries::of_element(
		w_n(5),
		5,
		&PURPLE,
		&|c, s, st| {
			EmptyElement::at(c)
				+ Circle::new((0, 0), s, st.filled())
				+ Text::new(
					format!("({:.2?}, {:.2?})", c.0, c.1),
					(15, 0),
					("sans-serif", 20).into_font(),
				)
		},
	))?;

	root.present()?;

	Ok(())
}
```
![](https://user-images.githubusercontent.com/72962900/228921832-c5bb93a0-a97f-4e2f-b7fe-78001b80c08b.png)

It looks great but I want both axes to be at the center so that it can show where is the origin.

I am using plotters `0.3.4`. If you need additional context, I can add more details later.

Thank you in advance for your help.

**Comment by Fabus1184** [comment]
Any updates on this? Seems like a very common use case to me...
